libraries/AP_Scripting/lua_repl.cpp

```diff
@@ -188,11 +213,26 @@ int lua_scripts::loadline(lua_State *L) {
     return status;
 }
 
-// push the tring into the terminal, blocks until it's queued
+// push the string into the terminal, blocks until it's queued
 void lua_scripts::terminal_print(const char *str) {
-    if ((AP::FS().write(terminal.output_fd, str, strlen(str)) == -1) ||
-        (AP::FS().fsync(terminal.output_fd) != 0)) {
-        terminal.session = false;
+    switch (terminal.type) {
+    case AP_Scripting::relp_type::REPL_FILE:
+    {
```

Comment:
I like scoping in general, but is there any reason you actually need it on either of these? I don't see anything being declared in either of the cases.